intellij-setup.md

```diff
@@ -28,23 +28,27 @@ Android Studio (and various other JetBrains editors) is currently not supported.
 
 ### Installing the plugins
 
-We provide two plugins:
+To use IntelliJ with Flutter, you must install two plugins:
```

Comment:
@mit-mit Is there a reason our users need to think about the Dart plugin at all? Could we just talk about the Flutter plugin and have it do the rest, including pulling in the Dart plugin?